cooking_test/Makefile

```diff
@@ -0,0 +1,52 @@
+BASE    = data/cooking
```

Comment:
I always forget about how Makefiles. Can help clean up tons of bash scripts littered all around! Nice addition here